src/engine/framework/System.cpp

```diff
@@ -51,6 +51,33 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <sys/file.h>
 #endif
 
+#if defined(DAEMON_USE_FPE)
+	#if defined(__USE_GNU) || defined (__APPLE__)
+		#include <cfenv>
+		#pragma FENV_ACCESS ON
+		#define DAEMON_USE_FPE_AVAILABLE
+	#elif defined(_MSC_VER)
+		#include <float.h>
+		#pragma fenv_access (on)
+		#define DAEMON_USE_FPE_AVAILABLE
+	#endif
+#endif
+
+#if defined(DAEMON_USE_FPE_AVAILABLE)
+	static Cvar::Cvar<bool> common_fpe_invalid("common.fpe.invalid",
+		"enable floating point exception for operation with NaN",
+		Cvar::TEMPORARY, false);
```

Comment:
If you want them to be settable only from the command line, you can use `Cvar::INIT`, which stops them from being changed later with an error message.

Reply:
Done.